examples/resource_lacework_integration_aws_eks_audit_log/audit_log_without_s3_arn/main.tf

```diff
@@ -0,0 +1,58 @@
+terraform {
+  required_providers {
+    lacework = {
+      source = "lacework/lacework"
+    }
+  }
+}
+
+variable "name" {
+  type    = string
+  default = "AWS EKS audit log integration example no S3"
+}
+
+variable "sns_arn" {
+  type      = string
+  default   = "arn:aws:sns:us-west-2:123456789123:foo-lacework-eks"
+}
+
+variable "s3_bucket_arn" {
+  type      = string
+  default   = "arn:aws:s3:::example-bucket-name"
+}
+
```

Comment:
Can remove this variable from the example.

Reply:
👍